src/components/renderer/renderer.tsx

```diff
@@ -44,6 +45,20 @@ export default class Editor extends React.Component<Props> {
       }
     }
 
+    if (this.props.export) {
+      const ext = this.props.renderer === 'canvas' ? 'png' : 'svg';
+      const url = view.toImageURL(ext);
+      url.then(function(url) {
+        var link = document.createElement('a');
+        link.setAttribute('href', url);
+        link.setAttribute('target', '_blank');
+        link.setAttribute('download', 'export.'+ ext);
```

Comment:
instead of downloading, can we just open it in a new tab?

Reply:
Actually, View API gives URI for `png` whose support is removed by many browsers in a new tab, they did it for security reasons. Now we have the only option of downloading it. Still, we can open SVG in a new tab.

Reply:
I think for SVG it's more important. My use case is that I want to print the SVG as PDF. It's not important for PNGs.